scripts.js

```diff
@@ -178,3 +180,54 @@ closePopupBtn.forEach((btn, index) => {
     popups[index].classList.remove("open-popup");
   });
 });
+// form validation
+const form = document.querySelector("form");
+const email = document.querySelector('input[type="email"]');
+const msg = document.querySelector("form .message");
+
+function showError(m) {
+  msg.style.display = "block";
+  msg.innerText = m;
+}
+
+function showSuccess(mass) {
+  msg.style.display = "block";
+  msg.style.color = "white";
+  msg.style.backgroundColor = "green";
+  msg.innerText = mass;
+}
+
+function checkLowerCase(input) {
+  if (input.value !== input.value.toLowerCase()) {
+    showError("please write the email in lowercase");
+  } else {
+    showSuccess("Thank You.");
+    form.submit();
+  }
+}
+
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+  checkLowerCase(email);
+});
+// local storage
+const valueInput = JSON.parse(localStorage.getItem("valueInput")) || [];
+
+function addVale(n, e, t) {
+  valueInput.push({ n, e, t });
+  localStorage.setItem("valueInput", JSON.stringify(valueInput));
```

Comment:
- [ ] Kindly store an `object` and not an `array` in the `localStorage`. You don't need the `valueInput` array. 👍 

#### Hint: `localStorage.setItem('valueInput', JSON.stringify({n, e, t}))`